src/store/tokens.ts

```diff
@@ -53,15 +60,18 @@ export const getters = getterTree(state, {
   getAllTokens(state): Tokens {
     return state.allTokens;
   },
-  getRestrictedTokens(state): Tokens {
-    return Object.fromEntries(Object.entries(state.allTokens).filter((e) => state.restrictedTokens.includes(e[1].symbol)));
+  getRestrictedTokens(state): TokenSymbol[] {
+    return state.restrictedTokens;
   },
   getAvailableTokens(state): Tokens {
     return Object.fromEntries(Object.entries(state.allTokens).filter((e) => !state.restrictedTokens.includes(e[1].symbol)));
   },
   getTokenPrices(state): ZkInTokenPrices {
     return state.tokenPrices;
   },
+  getTokenPriceTick(state): number {
+    return state.tokenPricesTick;
```

Comment:
Not sure what the tick is meant to, but if it is for updating token prices, then maybe we don't need it? We are not an exchange and our prices don't need to be 100% realtime accurate